startup/preinit

```diff
@@ -50,11 +50,7 @@ for arg in $args; do
       [ -z "$ramdisk_done" ] && /opt/xensource/installer/S05ramdisk start
       ramdisk_done=1
       copy=0;;
-    mount=*)
-      [ -z "$mount_done" ] && /opt/xensource/installer/S06mount start
```

Comment:
Don't you also need to remove the S06mount file?

Reply:
Well... more than needed it's cleaner, but it's easier from the packaging prospective as it does not need to update the `spec` file. That is I'll do it obviously.